templates/core/core-dpl.yaml

```diff
@@ -28,6 +28,34 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+      - name: "change-permission-of-directory"
+        image: {{ template "harbor.changePermissionImage" . }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        command: ["/bin/sh"]
+        args: ["-c", "chown -R 10000:10000 /etc/core || true"]
+        volumeMounts:
```

Comment:
Is there any pattern in k8s to avoid such duplication?  Similar to `volume-from`